iconbutton/internal/icon-button.ts

```diff
@@ -126,6 +130,15 @@ export class IconButton extends iconButtonBaseClass implements FormSubmitter {
 
   @state() private flipIcon = isRtl(this, this.flipIconInRtl);
 
+  @query('.link') private readonly linkElement!: HTMLElement | null;
```

Comment:
Update this to capture both the button and anchor element that could be rendered. Both can have clicks forwarded to them.

```ts
@query('.icon-button') private readonly buttonElement!: HTMLElement | null;
```